
[34m[1mwandb[39m[22m:   4 of 4 files downloaded.
Traceback (most recent call last):
  File "/Users/boisalai/GitHub/mlops-zoomcamp-2023/notebooks/wandb/train.py", line 74, in <module>
    run_train()
  File "/Users/boisalai/.local/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/boisalai/.local/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/boisalai/.local/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/boisalai/.local/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/boisalai/GitHub/mlops-zoomcamp-2023/notebooks/wandb/train.py", line 66, in run_train
    artifact.add_file("logistic_regression.pkl")
  File "/Users/boisalai/miniconda3/envs/mlops-zoomcamp-env/lib/python3.9/site-packages/wandb/sdk/wandb_artifacts.py", line 424, in add_file
    raise ValueError("Path is not a file: %s" % local_path)
ValueError: Path is not a file: logistic_regression.pkl